sinch/core/adapters/httpx_adapter.py

```diff
@@ -0,0 +1,60 @@
+import httpx
+from sinch.core.ports.http_transport import AsyncHTTPTransport, HttpRequest
+from sinch.core.endpoint import HTTPEndpoint
+from sinch.core.models.http_response import HTTPResponse
+
+
+class HTTPXTransport(AsyncHTTPTransport):
+    def __init__(self, sinch):
+        super().__init__(sinch)
+        self.http_session = None
+
+    async def request(self, endpoint: HTTPEndpoint) -> HTTPResponse:
+        request_data: HttpRequest = self.prepare_request(endpoint)
+        request_data: HttpRequest = await self.authenticate(endpoint, request_data)
+
+        if not self.http_session:
+            self.http_session = httpx.AsyncClient()
```

Comment:
The http_session is not properly closed and it can lead to resource leaks

Reply:
We've already discussed this on our weekly meeting. I'll mark this as resolved.